### ISSUE TITLE
PvP teamName returning "18"

### ISSUE THREAD

**Issue by t3rr11** [Bungie-net/api#627]
So i was just wondering if anyone else has been having this issue where PGCR PvP matches are returning the teamId in the teamName field. 

Examples here: 
https://www.bungie.net/Platform/Destiny2/Stats/PostGameCarnageReport/2100769930/
https://www.bungie.net/Platform/Destiny2/Stats/PostGameCarnageReport/2100679634/
https://www.bungie.net/Platform/Destiny2/Stats/PostGameCarnageReport/2100508348/

Bravo team returns normal: teamId: 17, teamName: Bravo.
Alpha team returns teamId: 18, teamName: 18.

**Comment by v-fedorov** [comment]
Or possibly, 
Alpha team returns: teamId: 17, teamName: Bravo.
Bravo team returns teamId: 18, teamName: 18.

heres a sample screenshot from a game (pgcr 2091262861) that says Bravo won, team 18 lost

![image](https://user-images.githubusercontent.com/1764879/44920640-793d2d80-ad0e-11e8-9333-5f281f7692a3.png)

Im just not sure what the icons are in-game lol

**Comment by ashleypb** [comment]
Yeah v-fedorov has it right.

Alpha team returns: teamId: 17, teamName: Bravo.
Bravo team returns teamId: 18, teamName: 18.

**Comment by vthornheart-bng** [comment]
TFS 706921 to investigate

**Comment by karandashek** [comment]
Hi guys!
Do you have any update for this? it seems API still returns "18" as name :)

**Comment by t3rr11** [comment]
Yeah still got the same issue, haven't quite worked out a way around it, if I adapt to the new change it breaks the old records and if I don't then it displays the wrong teams. I kinda just closed my project for now.

**Comment by ArkahnX** [comment]
@vthornheart-bng can we get an update on the status of this issue?

**Comment by vthornheart-bng** [comment]
I haven't had a chance to investigate - I apologize!  As Gandalf once said; "I was... delayed"

I'll see if I can bump up the priority on this one.

**Comment by t3rr11** [comment]
I'd like to bring this up again, i've started to rework my activities tracking page and noticed that: 

Alpha team still returns: teamId: 17, teamName: Bravo.
Bravo team still returns teamId: 18, teamName: 18.

![image](https://user-images.githubusercontent.com/2139233/60072586-2204bf80-975d-11e9-89c1-f0601c5ca681.png)
